pkg/cmd/delete.go

```diff
@@ -85,10 +92,6 @@ func NewCompDeleteCommand(c types.Args, ioStreams cmdutil.IOStreams) *cobra.Comm
 			return errors.New("must specify name for the app")
```

Comment:
```suggestion
			return errors.New("must specify name for the component")
```

We should change the error message

Reply:
```go
if len(args) < 1 {
			return errors.New("must specify name for the app")
		}
```
the `args` get the name of component not app , app name in flag `-a`